src/routes/README.md

```diff
@@ -0,0 +1,6 @@
+# Routes
+
+There are two shared routes as defined in the [oauth2 router](./oauth2/index.js):
+
+- `/oauth2/authorize` - entry point for the CRI, responsible for initialising the session on backend, and then redirecting into the first page of the CRI
+- `/callback` - exit point for the CRI, responsible for redirecting back to the caller
```

Comment:
Do you think it could be helpful for someone coming in cold to give a bit of information about using `routes` here but HMPO `steps` for routing in the CRIs themselves?? I know this has caused some confusion for me before

Reply:
It would definitely be useful to have some clear documentation across the programme as to how the Design System and Express works together to render full pages. Related to that there's an additional piece of documentation that would be useful describing how the extra layer of the `hmpo-form-wizard` and `hmpo-components` should be used.

I'm not sure where that should live though - this repository has 1 usage of `hmpo-form-wizard`, in a library that should be extracted out into a [separate test package](https://github.com/govuk-one-login/ipv-cri-common-express/issues/405). There's no usage of the `steps` in this repo at all. There is a small usage of the `hmpo-component` which are used as base templates.

Perhaps the explanation should be a separate document and a separate PR?